src/pages/Login/LoginPage.jsx

```diff
@@ -26,18 +26,28 @@ export default function LoginPage() {
       await loginUser(credentials).unwrap();
       navigate(ROUTES.HOME);
     } catch (err) {
-      console.error(AUTH_ERRORS.LOGIN_ERROR, err);
+      setModalData({
+        title: AUTH_ERRORS.LOGIN_FAILED,
+        message: extractApiError(err, AUTH_ERRORS.GENERAL_LOGIN_ERROR),
+        onClose: () => setModalData(null),
+      });
     }
   };
 
   return (
     <div>
       <LoginForm onSubmit={handleLogin} isLoading={isLoading} />
-      {error && (
-        <SharedTypography variant="body2" color="error">
-          {extractApiError(error, AUTH_ERRORS.GENERAL_LOGIN_ERROR)}
-        </SharedTypography>
+
+      {modalData && (
+        <StatusModal
+          open={!!modalData}
```

Comment:
There’s no need to check modalData twice.
Since you’re already conditionally rendering the StatusModal using modalData && (...), passing open={!!modalData} is redundant.

```suggestion
        <StatusModal
          open
```